modules/customer-magento/repositories/useReorderItemsRepository.ts

```diff
@@ -0,0 +1,27 @@
+import useMagento from '#ioc/composables/useMagento'
+import ReorderItems from '#ioc/graphql/mutations/ReorderItems'
+import ToCart from '#ioc/mappers/ToCart'
+
+export default () => {
+  const magento = useMagento()
+
+  return async (
+    orderNumber: string,
+  ): Promise<{
+    cart: ReturnType<typeof ToCart>
+  }> => {
+    const { data } = await magento.graphql(
+      ReorderItems().with({
+        orderNumber,
+      }),
+    )
+
+    if (data?.reorderItems?.user_errors?.length > 0) {
```

Comment:
This doesn't seem right. Graphql field is called, `userInputErrors`, here you have `user_errors` and in the if is `userErrors`. BTW same problem is in the `modules/wishlist-magento/repositories/useAddToWishlistRepository.ts` so fix that also, please.

Reply:
Hi, you are right with Reorderitems the user errors come in userInputErros, however in the case of AddProductsToWishlist the come in the field user_errors according to https://magexo.atlassian.net/browse/SFX-22.